scripts/static-binaries/build-on-ubuntu.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# Build concordium-node and related binaries in a format suitable for distribution.
+# The binaries are linked mostly statically, apart from a few common libraries that
+# seem difficult to impossible to link statically. These are libc, libpq.
+
+set -euxo pipefail
+
+# The script must be invoked with the concordium-node source directory as working directory.
+
+# Expected parameters:
+# - EXTRA_FEATURES, features to use in addition to `static` when building the node, e.g., collector. Comma separated list.
+
+extra_features="$EXTRA_FEATURES"
```

Comment:
As the script doesn't take any args, you could use `$@` to take the args as extra features. As you have `set -u` this setup will crash if `EXTRA_FEATURES` isn't set. You could of course also use `${EXTRA_FEATURES:-""}`.

Reply:
> As the script doesn't take any args, you could use `$@` to take the args as extra features. As you have `set -u` this setup will crash if `EXTRA_FEATURES` isn't set. You could of course also use `${EXTRA_FEATURES:-""}`.

I'm not sure I agree. The documentation explicitly states that EXTRA_FEATURES is an expected parameter, so crashing if it isn't set should be expected. 

I don't really see adding a default value for it materially helps anything.

Reply:
I see that the documentation in the list above says this is optional, as opposed to the documentation in the script itself. I'll rectify the discrepancy.